### PR TITLE
Support styling mnrva 529

### PR DESCRIPTION
**Pull request by lovedeep92**
**JIRA:**
See ticket here - [](https://jira.rax.io/browse/MNRVA-529)

**Description:**
Add Support for Styling to Hedwig.

**1)** Add separate css folder inside src and create main.css file.

**2)** Bundle stylesheet (main.css) using posts rollup plugin.

**3)** Add common css property into main.css file and just import that file into webcomponent js file. 

**Screenshots:**
(if applicable)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "@babel/core": "7.5.4",
     "babelrc-rollup": "3.0.0",
     "backstopjs": "5.0.1",
+    "cssnano": "^4.1.10",
```
Comment (FarajiA) [inline package.json:30]: Let's do absolute versions for our packages.

File: package.json [hunk 2]
```diff
@@ -40,12 +41,16 @@
...
+    "postcss-cssnext": "^3.1.0",
+    "postcss-nested": "^4.2.3",
+    "postcss-simple-vars": "^5.0.2",
     "puppeteer": "1.18.1",
     "rollup": "1.17.0",
     "rollup-plugin-babel": "4.3.3",
     "rollup-plugin-commonjs": "9.1.0",
     "rollup-plugin-json": "3.1.0",
     "rollup-plugin-node-resolve": "3.0.0",
+    "rollup-plugin-postcss": "^3.1.3",
```
Comment (FarajiA) [inline package.json:53]: Nice! Rollup has a plugin for this.

File: package.json (hunk 1 above)
Comment (lovedeep92) [reply package.json]: yeah and its done !!

File: package.json (hunk 2 above)
Comment (lovedeep92) [reply package.json:53]: yeah .. it has all these dependencies!!

Comment (FarajiA) [reply package.json:53]: Forgot one.

**Pull request merged**